skyportal/utils/offset.py

```diff
@@ -84,6 +118,7 @@ def get_url(*args, **kwargs):
         return None
 
 
+@offsets_memory.cache
```

Comment:
Would be useful if this was also made available to other web workers

Reply:
Beyond this scope here. This would require cache directories which are shared across workers.